### DISCUSSION TITLE
Two identical simulation - different attempt - 100% same results?

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#13015] [Q&A] [answered]
Dear all,

as I'm trying to understand an issue previously faced in another post, I'm running a simulation I've just run to compare results among FDS versions.

I'm using last FDS 6.9 one for Pyrolisis, but due to the fact I must also prove a safe egress, I will need to use version 6. 7.7 for the final simulation which should be the latest supporting FDS+EVAC according to the guy who sold me the GUI. Should I trust him? ;) 

I was just wondering if given the same fds code, the same fds software version and the same PC, shall I get 100% the same identical output file? I guess that this is impossible due to the fact that floating errors cannot be predicted. However how much the .out, the hrr.csv files might differs?

**Comment by drjfloyd** [comment]
You will not get the exact same answer, and you if you are predicting pyrolysis rather than specifying the heat release rate there could be large differences.

**Reply by LordMicidial** [reply]
To make it simple and close the question, are you making differences between 9.1 ( Specified Heat Release Rate) and 9.2 (Complex Pyrolysis Models) of the user guide? am I right? 

With Specified Heat Release Rate i should get similar answer.

**Reply by drjfloyd** [reply, marked as answer]
For a specified fire would generally expect answers to be similar unless there is something about the simulation that triggers some bug that has been fixed between 6.7.7 and 6.9.1.

**Reply by TristanHehnen** [reply]
I would not expect that two different FDS versions lead to the same results. From 6.7.0 to now, there have been over 4300 changes to FDS. General consistency is assessed with the validation cases, but still the results are not identical. See for example section "6.1.5 NIST Pool Fires" in the validation guide across different versions: 

![grafik](https://github.com/firemodels/fds/assets/34650649/25a5efce-c91b-4be2-abdc-179c2ab6444a)

Maybe you could run validation cases that are capturing the primary phenomenon you are interested in and compare the results. With that you could get some idea how the results change. And specifically you can compare against the data from the respective experiment.

For your particular case, it may be noteworthy that the default formula for `SOOT` has changed in 6.8.0. to be now pure carbon. This has an impact on the enthalpies and may have an effect on your simulation. For example, if you prescribe the mass of combustible hydrocarbons that is to be released into the domain. 

Personally, I strongly recommend to stay with the same computer, operating system and FDS version for a given project, if at all possible. Otherwise, chances are high to loose internal consistency within the project. 
What different results mean for the same case across different FDS versions is in my view a completely different question. 
The question becomes how same is "the same"? Again, 4k+ changes...

**Reply by TristanHehnen** [reply]
So, broadly speaking: the same answer as was already provided above... 😁
